Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/jupyter_cache/executors/utils.py", line 51, in single_nb_execution
    executenb(
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/client.py", line 1117, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/util.py", line 78, in wrapped
    return just_run(coro(*args, **kwargs))
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/util.py", line 57, in just_run
    return loop.run_until_complete(coro)
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/asyncio/base_events.py", line 616, in run_until_complete
    return future.result()
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/client.py", line 558, in async_execute
    await self.async_execute_cell(
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/client.py", line 862, in async_execute_cell
    self._check_raise_for_error(cell, exec_reply)
  File "/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/nbclient/client.py", line 765, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
engine = create_engine(
    "omnisci://demouser:HyperInteractive@"
    "metis.mapd.com:443/mapd?protocol=https"
)

con = engine.connect()
------------------

[0;31m---------------------------------------------------------------------------[0m
[0;31mNoSuchModuleError[0m                         Traceback (most recent call last)
[0;32m/tmp/ipykernel_2606/2237949546.py[0m in [0;36m<module>[0;34m[0m
[0;32m----> 1[0;31m engine = create_engine(
[0m[1;32m      2[0m     [0;34m"omnisci://demouser:HyperInteractive@"[0m[0;34m[0m[0;34m[0m[0m
[1;32m      3[0m     [0;34m"metis.mapd.com:443/mapd?protocol=https"[0m[0;34m[0m[0;34m[0m[0m
[1;32m      4[0m )
[1;32m      5[0m [0;34m[0m[0m

[0;32m<string>[0m in [0;36mcreate_engine[0;34m(url, **kwargs)[0m

[0;32m/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/sqlalchemy/util/deprecations.py[0m in [0;36mwarned[0;34m(fn, *args, **kwargs)[0m
[1;32m    296[0m                         [0mstacklevel[0m[0;34m=[0m[0;36m3[0m[0;34m,[0m[0;34m[0m[0;34m[0m[0m
[1;32m    297[0m                     )
[0;32m--> 298[0;31m             [0;32mreturn[0m [0mfn[0m[0;34m([0m[0;34m*[0m[0margs[0m[0;34m,[0m [0;34m**[0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    299[0m [0;34m[0m[0m
[1;32m    300[0m         [0mdoc[0m [0;34m=[0m [0mfn[0m[0;34m.[0m[0m__doc__[0m [0;32mis[0m [0;32mnot[0m [0;32mNone[0m [0;32mand[0m [0mfn[0m[0;34m.[0m[0m__doc__[0m [0;32mor[0m [0;34m""[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/sqlalchemy/engine/create.py[0m in [0;36mcreate_engine[0;34m(url, **kwargs)[0m
[1;32m    532[0m     [0mu[0m[0;34m,[0m [0mplugins[0m[0;34m,[0m [0mkwargs[0m [0;34m=[0m [0mu[0m[0;34m.[0m[0m_instantiate_plugins[0m[0;34m([0m[0mkwargs[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    533[0m [0;34m[0m[0m
[0;32m--> 534[0;31m     [0mentrypoint[0m [0;34m=[0m [0mu[0m[0;34m.[0m[0m_get_entrypoint[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    535[0m     [0mdialect_cls[0m [0;34m=[0m [0mentrypoint[0m[0;34m.[0m[0mget_dialect_cls[0m[0;34m([0m[0mu[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    536[0m [0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/sqlalchemy/engine/url.py[0m in [0;36m_get_entrypoint[0;34m(self)[0m
[1;32m    641[0m         [0;32melse[0m[0;34m:[0m[0;34m[0m[0;34m[0m[0m
[1;32m    642[0m             [0mname[0m [0;34m=[0m [0mself[0m[0;34m.[0m[0mdrivername[0m[0;34m.[0m[0mreplace[0m[0;34m([0m[0;34m"+"[0m[0;34m,[0m [0;34m"."[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0;32m--> 643[0;31m         [0mcls[0m [0;34m=[0m [0mregistry[0m[0;34m.[0m[0mload[0m[0;34m([0m[0mname[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[0m[1;32m    644[0m         [0;31m# check for legacy dialects that[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m
[1;32m    645[0m         [0;31m# would return a module with 'dialect' as the[0m[0;34m[0m[0;34m[0m[0;34m[0m[0m

[0;32m/usr/share/miniconda3/envs/sqla-omnisci/lib/python3.8/site-packages/sqlalchemy/util/langhelpers.py[0m in [0;36mload[0;34m(self, name)[0m
[1;32m    341[0m                 [0;32mreturn[0m [0mimpl[0m[0;34m.[0m[0mload[0m[0;34m([0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    342[0m [0;34m[0m[0m
[0;32m--> 343[0;31m         raise exc.NoSuchModuleError(
[0m[1;32m    344[0m             [0;34m"Can't load plugin: %s:%s"[0m [0;34m%[0m [0;34m([0m[0mself[0m[0;34m.[0m[0mgroup[0m[0;34m,[0m [0mname[0m[0;34m)[0m[0;34m[0m[0;34m[0m[0m
[1;32m    345[0m         )

[0;31mNoSuchModuleError[0m: Can't load plugin: sqlalchemy.dialects:omnisci
NoSuchModuleError: Can't load plugin: sqlalchemy.dialects:omnisci

